.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
```

Comment:
Sorry, I wonder, shouldn't this `CONTRIBUTNG.md` be outside folder `.github`?  Otherwise, someone just browsing the github repo will never notice it.

The one from Magpie is  outside `.github` folder https://github.com/Ouranosinc/Magpie/blob/master/CONTRIBUTING.rst

Also if you move it outside, make it `.rst`, not `.md`, then you can include it in ReadTheDocs as well, probably in file https://github.com/bird-house/birdhouse-deploy/blob/master/docs/source/index.rst

Reply:
@tlvu it's done but I had a hard time to render the image both in github and the doc. Right now the image reference is relative to the doc source directory which make sphinx happy, but github cannot renders it correctly while browsing the CONTRIBUTING.rst. If I make the path relative to the root directory github is happy but not sphinx. If someone knows how to fix this, I will be more than happy to add the fix.

Reply:
I used an absolute url and it should work.

Reply:
Just thought about this, using absolute url to the image, if someone update the image in a PR, browsing that PR will still show the image on `master` branch.  Something to keep in mind ...

Reply:
I know, this is a drawback (but I don't think this image will change a lot). If you have a better solution go ahead. 
Francis has written a sphinx extension to handle rst references in the Magpie documentation, but I don't think anybody has half a day to implement an extension for 1 image that may never be uptaded.

Reply:
> I don't think this image will change a lot

Agreed.  I wasn't saying this is deal breaker.  Was just saying it's something to keep in mind if ever a PR touch that image.  Sorry if I sent the wrong impression.